Example/.gitignore

```diff
@@ -54,3 +54,6 @@ buck-out/
 
 # Bundle artifact
 *.jsbundle
+
+# metro-with-symlinks
```

Comment:
This makes developing while experimenting with the examples a lot easier. With this you can use `yarn link`.